clang/lib/CConv/ProgramInfo.cpp

```diff
@@ -481,10 +479,10 @@ void ProgramInfo::exitCompilationUnit() {
 bool
 ProgramInfo::insertIntoExternalFunctionMap(ExternalFunctionMapType &Map,
                                            const std::string &FuncName,
-                                           std::set<FVConstraint *> &ToIns) {
+                                           const std::set<FVConstraint *> &ToIns) {
   bool RetVal = false;
   if (Map.find(FuncName) == Map.end()) {
-    Map[FuncName] = ToIns;
+    Map.insert(std::make_pair(FuncName, ToIns));
```

Comment:
Why this change? Is it because you've made something `const` ? Does it allow a map to be treated as a relation?

Reply:
I think I can undo this change. This was required after introducing `const`s because I had `map<string, const set<ConstraintVariable *>>` and  `operator[]` is not defined for `const` value type. Since the map is now  `map<string, ConstraintVariable *>`, I shouldn't need it anymore.